tests/installation/bootloader_zkvm.pm

```diff
@@ -56,6 +56,10 @@ sub run() {
             $cmdline .= " autoyast=" . data_url(get_var('AUTOYAST')) . " ";
         }
 
+        if (get_var('FIPS')) {
+            $cmdline .= "fips=1 ";
+        }
```

Comment:
I don't understand. The call to "specific_bootmenu_params" just two lines - see `lib/bootloader_setup.pm` below should already achieve the same. Is it not working?